packages/pages/src/dev/dev.ts

```diff
@@ -35,6 +44,12 @@ export const devCommandModule: CommandModule<unknown, DevArgs> = {
         type: "boolean",
         demandOption: false,
         default: true,
+      })
+      .option("open-browser", {
+        describe: "Open the browser",
```

Comment:
I had just put up a placeholder description before. Can we change this to something like `Automatically opens the browser on server start-up`?